boot/freeldr/freeldr/lib/fs/fs.c

```diff
@@ -167,6 +242,9 @@ ARC_STATUS ArcOpen(CHAR* Path, OPENMODE OpenMode, ULONG* FileId)
     if (!DeviceName)
         return ENOMEM;
 
+ERR("ArcOpen: Path(0x%p): '%s' ; DeviceName(0x%p): '%.*s' ; FileName: '%s'\n",
```

Comment:
Inconsistent indentation?

Reply:
yep, but don't worry, this is a temporary trace that I'll remove.